snuba/migrations/snuba_migrations/querylog/0002_status_type_change.py

```diff
@@ -0,0 +1,47 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Array, Column, Enum, LowCardinality, String
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+
+
+class Migration(migration.MultiStepMigration):
+    """
+    Drops the status enum and replaces it with a LowCardinality string
+    now that the support for low cardinality strings is better.
+    """
+
+    blocking = False
```

Comment:
I assume that this operation will take a long time to run since all the values in the status column would be rewritten. So blocking = True.